omim2obo/utils/utils.py

```diff
@@ -21,12 +22,32 @@ def remove_angle_brackets(uris: Union[str, List[str]]) -> Union[str, List[str]]:
     return uris2[0] if str_input else uris2
 
 
-def get_d2g_exclusions_by_curator(path=DISEASE_GENE_EXCLUSIONS_PATH) -> Dict[str, Optional[str]]:
-    """Get disease-gene exclusions
+def get_d2g_config_by_curator(path: str) -> Dict[str, Optional[URIRef]]:
+    """Get information for manually curated disease-gene associations
 
     :return: Dict[str, str]: Phenotype MIM as keys, ORCID of curator as values
     """
     df = pd.read_csv(path, sep='\t').fillna('')
     df['phenotype_mim'] = df['omim_id'].apply(lambda x: x.split(':')[1])
     phenotype_mim_orcid_map = {x['phenotype_mim']: x['orcid'] for x in df.to_dict(orient='records')}
     return {k: ORCID[v] if v else None for k, v in phenotype_mim_orcid_map.items()}
+
+
+def get_d2g_protections_by_curator(path=DISEASE_GENE_PROTECTIONS_PATH) -> Dict[str, Optional[URIRef]]:
```

Comment:
Overall, try to use the word protected vs. protections.

Reply:
Renamed and pushed along w/ the other changes.